### ISSUE TITLE
Save config not working

### ISSUE THREAD

**Issue by simonmaass** [mageplaza/magento-2-delete-orders#14]
### Preconditions (*)

1. Magento 2.4.1
2. Module Version 1.2.1

### Steps to reproduce (*)

2. Enable Module
3. Set Schedule For to "Disable"

### Expected result (*)

1. New config saved

### Actual result (*)

![image](https://user-images.githubusercontent.com/32599710/108527842-0acff980-72d3-11eb-9b00-b436ec6c0dab.png)

I belive this doesnt work:

```
$time      = $this->getData('groups/schedule/fields/time/value');
        $frequency = $this->getData('groups/schedule/fields/schedule_for/value');

        if ($frequency !== ValueConfig::DISABLE) {
            $cronExprArray = [
                (int) $time[1], //Minute
                (int) $time[0], //Hour
                $frequency === \Magento\Cron\Model\Config\Source\Frequency::CRON_MONTHLY ? '1' : '*', //Day of the Month
                '*', //Month of the Year
                $frequency === \Magento\Cron\Model\Config\Source\Frequency::CRON_WEEKLY ? '1' : '*', //Day of the Week
            ];
```

groups/schedule/fields/time/value has value "00,00,00"

**Comment by Victor-Mageplaza** [comment]
I'm Victor from Mageplaza. I would like to answer your question.
Maybe _**$frequency**_ does not int variable. You should check this variable type
Thank you!

**Comment by simonmaass** [comment]
this config key "groups/schedule/fields/schedule_for/value" doesnt always exists

**Comment by phamcuongmp** [comment]
Please help me download and apply the file below:
[file patch](https://drive.google.com/file/d/1BrJeCnwlAvn92fO4lJBo54dYW4BcQo77/view?usp=sharing)
vendor/mageplaza/module-delete-orders//Model/Config/Backend/Order/Frequency.php

Let me know about the problem.

We have updated the code. The error will be fixed on the next version of the Delete Order extension.
Thank you so much.

**Comment by simonmaass** [comment]
@phamcuongmp great - thank you. When will you release the new version?

**Comment by phamcuongmp** [comment]
@simonmaass Yes. We will release the new version shortly.

**Comment by simonmaass** [comment]
@phamcuongmp any chance to release the new version?

**Comment by phamcuongmp** [comment]
@simonmaass The Delete Order version 4.0.0 has been released for Magento 2.4. You totally can download and install it now!
